src/main/scala/chisel3/util/Arbiter.scala

```diff
@@ -10,28 +10,15 @@ import chisel3.internal.naming.chiselName  // can't use chisel3_ version because
 
 /** IO bundle definition for an Arbiter, which takes some number of ready-valid inputs and outputs
   * (selects) at most one.
-  * @groupdesc Signals The actual hardware fields of the Bundle
```

Comment:
Okay thanks 
I think this line 13 should be replaced
`* @groupdesc Signals The actual hardware fields of the Bundle`

Reply:
Let me get this, so you are suggesting adjusting the files instead of removing them?  I thought I was supposed to remove them since they are not part of this PR @mwachs5, what do you think?

Reply:
Yes because I think if you remove them completely, it give you errors.
So I think something you can do is to revert the changes on this files but I don't know exactly when you did the changes and with what commit you did them.
So that's why I think it will be better to do them manually.
I also think the the what you did during the cleaning up was recovering the Arbiter.scala file that was deleted, so I think that means deleting the files will not be safe.

Reply:
@Burnleydev1 I wasn't planning on deleting them though... I will try what you said. Thank you.

Reply:
@shorla, sorry I misquoted you, I hope this works.

Reply:
@Burnleydev1 noo... It is fine and I really appreciate your help and how you always try to help out. Thank you.

Reply:
@Shorla, I’m happy to hear that🙏🏾.

Reply:
Hi @Shorla, I wasn't able to do the changes because I don't have the right to do so. so I think you have to put those changes manually.

Reply:
That's all I can do is point to the location and I'm not allowed to make the actual changes.

Reply:
@Burnleydev1 I am making them manually. I will push them once I am done. Thank you